src/suvela/Exam3.java

```diff
@@ -0,0 +1,46 @@
+package suvela;
+
+public class Exam3 {
+	int count;
+
+	void duplicateElement(int[] arr) {
+		for (int i = 0; i < arr.length; i++) {
+			count = 0;
+			int no = arr[i];
+			for (int j = 0; j < arr.length; j++) {
+				if (no == arr[j]) {
+					count++;
+					arr[i] = 0;
+				}
+
+			}
+			if (count > 1 && arr[i] == 0)
+				System.out.println("Duplicate elements are:" + no);
+		}
+	}
+
+	void duplicateString(String[] s) {
+		for (int i = 0; i < s.length; i++) {
+			count = 0;
+			String s1 = s[i];
+			for (int j = 0; j < s.length; j++) {
+				if (s1 == s[j]) {
```

Comment:
string should not be compared with == .
you should use .equals method

Reply:
changed.